server.go

```diff
@@ -70,6 +95,24 @@ func Handler(sink *lager.ReconfigurableSink) http.Handler {
 			sink.SetMinLevel(lager.FATAL)
 		}
 	}))
+	mux.Handle("/block-profile-rate", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		_rate, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			return
+		}
+		
+		rate, err := strconv.Atoi(string(_rate))
+		if err != nil {
+			fmt.Println(err)
```

Comment:
Since this is an http request, it's more likely to be helpful for the requester to see the error, would you mind changing this to a 400 error (with `w.WriteHeader(http.StatusBadReqeust)`) and then writing the error contents to the body? I think it's less likely that someone would be watching stdout when trying to reconfigure this on the fly.